euphonic/ureg.py

```diff
@@ -0,0 +1,17 @@
+from importlib.resources import files
```

Comment:
Do probably want a docstring here, given we're introducing the module in this PR.

Reply:
I've never used `importlib.resources`, what is the advantage over a `pathlib.Path`?

Reply:
It's based on the package structure rather than directly sniffing around the filesystem. I think that makes it a bit more robust for eggs, editable installs etc. Anyway it's the recommended way of handling data.

Reply:
Does that mean it needs a `__init__.py` in the data folder?

Reply:
Presumably: there is one. Haven't though about it for a while 😆

Reply:
> Do probably want a docstring here, given we're introducing the module in this PR.

done!